fastqpreprocessing/src/metricgatherer.h

```diff
@@ -0,0 +1,276 @@
+#ifndef __METRIC_GATHERER__
+#define __METRIC_GATHERER__
+/**
+ *  @file   metricgatherer.h
+ *  @brief  functions for file processing
+ *  @author Kishori Konwar
+ *  @date   2021-08-11
+ ***********************************************/
+
+#include <unordered_map>
+#include <map>
+#include <string>
+#include <regex>
+#include <iostream>
+#include <vector>
+#include <assert.h>
+#include <fstream>
+#include <iomanip>
+#include <math.h>
+#include "datatypes.h"
+
+using namespace std;
+
+//    Implementation of Welford's online mean and variance algorithm
```

Comment:
Should this be a multiline comment instead of many single line comments?

Reply:
yes, changed it now.